src/display/api.js

```diff
@@ -650,6 +650,16 @@ var PDFDocumentProxy = (function PDFDocumentProxyClosure() {
  */
 
 /**
+ * Page annotation parameters.
+ *
+ * @typedef {Object} GetAnnotationsParameters
+ * @param {string} intent - Determines which annotations that will be fetched,
```

Comment:
Nit: change this to either "Determines the annotations that will be fetched" or "Determines which annotations will be fetched"